### ISSUE TITLE
RadioButtonGroupView in List View

### ISSUE THREAD

**Issue by AhmedAdelGadElkareeem** [enisn/Xamarin.Forms.InputKit#133]
## My Question Here ?
i use RadioButtonGroupView in list view but i can select all radio buttons not only one

### Also I tried that:
Ad sit ut blandit dolore rebum dolore erat velit gubergren invidunt rebum ipsum stet amet takimata voluptua facilisi magna aliquyam kasd est et et eirmod diam sea invidunt sea duo ipsum illum invidun.

```xml
  <input:RadioButtonGroupView>
                 <StackLayout Orientation="Horizontal" Margin="0,10,0,5">
                                 <Label Text="{Binding HelpOrderName}" TextColor="Black" 
                                            HorizontalOptions="FillAndExpand"
                                                   FontSize="13" FontFamily="serif" VerticalOptions="Center"/>
                                 <input:RadioButton Color="#057482" CircleColor="#057482" 
                                                                               VerticalOptions="Center"/>
             </StackLayout>
</input:RadioButtonGroupView>
```

```csharp
   var myButton = new Button();
   myButton.Color = Color.Blue;
```

**Comment by AhmedAdelGadElkareeem** [comment]
<ListView x:Name="OrdersListView" HasUnevenRows="True" Margin="10,5,10,0"
                          SeparatorVisibility="None" HeightRequest="200">
                    <ListView.ItemTemplate>
                        <DataTemplate>
                            <renders:ExtendedViewCell SelectedBackgroundColor="White">
                                <StackLayout Margin="10,0,10,0">
                                    <input:RadioButtonGroupView>
                                        <StackLayout Orientation="Horizontal" Margin="0,10,0,5">
                                            <Label Text="{Binding HelpOrderName}" TextColor="Black" 
                                                   HorizontalOptions="FillAndExpand"
                                                   FontSize="13" FontFamily="serif" VerticalOptions="Center"/>
                                            <input:CheckBox Color="#057482" BorderColor="#057482" VerticalOptions="Center"/>
                                        </StackLayout>
                                    </input:RadioButtonGroupView>
                                    <BoxView HeightRequest="1" HorizontalOptions="FillAndExpand"
                                             Color="Black" Margin="10,0,10,0"/>
                                </StackLayout>
                            </renders:ExtendedViewCell>
                        </DataTemplate>
                    </ListView.ItemTemplate>
                </ListView>

**Comment by Plaba** [comment]
Hello @AhmedAdelGadElkareeem,
I think the problem is that you're putting each RadioButton in it's own RadioButtonGroupView (RBGV). Only one RadioButton can be selected per RBGV, but if you have multiple RBGV's, nothing prohibits all the RadioButtons from being selected. 

To fix your problem, you need to put all of the RadioButtons in one RBGV.

I hope I'm being clear,
--Plaba

**Comment by enisn** [comment]
> Hello @AhmedAdelGadElkareeem,
> I think the problem is that you're putting each RadioButton in it's own RadioButtonGroupView (RBGV). Only one RadioButton can be selected per RBGV, but if you have multiple RBGV's, nothing prohibits all the RadioButtons from being selected.
> 
> To fix your problem, you need to put all of the RadioButtons in one RBGV.
> 
> I hope I'm being clear,
> --Plaba

Yeah, @Plaba  is right. If you need generate radio buttons dynamicly you can use **SelectionView**
There is a docuement about it:
- [Adding dynamicly Radio Buttons](https://github.com/enisn/Xamarin.Forms.InputKit/wiki/Adding-dynamicly-RadioButtons)

**Comment by AhmedAdelGadElkareeem** [comment]
Already i put all radio buttons in one Radio group view check my Call code
added above you will see what i mean
